### PR TITLE
feat: add unlink command

### LINKED ISSUE #17: Add unlink command

### PR DESCRIPTION
**Pull request by DSpeckhals**
This commit adds the `unlink` command, which removes all symlinks defined in `bombadil.toml`. It also adds a corresponding section to the README.

Also a link to another example dotfiles repository is added to the README.

Fixes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -479,11 +479,20 @@ zsh_env = { source = "zsh/zshenv", target = ".zshenv" }
 starship = { source = "zsh/starship.toml", target = ".config/starhip.toml" }
 ```
 
+## Unlinking
```
Comment (oknozor) [inline README.md:482]: This sould be referenced in the toc

File: src/bin/bombadil.rs [hunk 2]
```diff
@@ -65,6 +66,9 @@ fn main() {
                 .takes_value(true)
                 .multiple(true)
                 .required(false)))
+        .subcommand(SubCommand::with_name(UNLINK))
```
Comment (oknozor) [inline src/bin/bombadil.rs:69]: `subcommand` closing parenthesis should be after `about`, currently the help message is not displayed correctly.

**Review by oknozor — CHANGES_REQUESTED** [review]

**Comment by oknozor** [comment]
@DSpeckhals  I forgot to mention, you can add your github handle to the author list in `cog.toml` :

```toml
authors = [
    { signature = "Paul Delafosse", username = "oknozor" },
    { signature = "Dustin Speckhals" = "DSpeckhals" },
]
```

This will be used to generate a link to your github profile in the next changelog.

File: src/bin/bombadil.rs (hunk 2 above)
Comment (DSpeckhals) [reply src/bin/bombadil.rs]: Ahhh...thanks for catching that!

**Review by oknozor — APPROVED** [review]

**Comment by oknozor** [comment]
Well done @DSpeckhals :champagne: !

**Pull request merged**